src/TLuaInterpreter.cpp

```diff
@@ -3204,17 +3204,16 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
 }
 
 // No documentation available in wiki - internal function
-void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool updateEvent)
+void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool resizeMudlet)
 {
     Host& host = getHostFromLua(L);
-    // position: 0 = top, 1 = right, 2 = bottom, 3 = left
     switch(position) {
-        case 0: host.mBorderTopHeight = size; break;
-        case 1: host.mBorderRightWidth = size; break;
-        case 2: host.mBorderBottomHeight = size; break;
-        case 3: host.mBorderLeftWidth = size; break;
+        case Qt::TopSection: host.mBorderTopHeight = size; break;
```

Comment:
Small bug, https://doc.qt.io/qt-5/qt.html#WindowFrameSection-enum is not 0 it is 3